src/components/navigation-header/index.tsx

```diff
@@ -61,7 +61,7 @@ const NavigationHeader: React.FC = () => (
             </li>
           ))}
         </ul>
-        <HeaderSearchInput />
+        <HeaderSearchInput theme="dark" />
```

Comment:
Previously this component inherited `color` from `navigation-header`. Have implemented a super basic `theme` within `header-search-input` to make things a lil more encapsulated.